workshop/content/english/30-python/70-advanced-topics/200-pipelines/3000-new-pipeline.md

```diff
@@ -9,7 +9,7 @@ Now we are ready to define the basics of the pipeline.
 
 Return to the file `pipeline_stack.py` and edit as follows:
 
-{{<highlight python "hl_lines=13 18-30">}}
+{{<highlight python "hl_lines=12 17-29">}}
```

Comment:
The differences I see are on lines `5 17-29` aside from the differences in formatting on lines 13-16. Where are you getting 12 from exactly?

Additionally when reviewing this, I noticed on the [previous page](https://cdkworkshop.com/30-python/70-advanced-topics/200-pipelines/2000-create-repo.html) that there's a dangling comma after the codecommit import line, as well as the dangling comma on _this_ page after the pipelines import in both the english and JA workshops

Reply:
Oops, it seems I made a mistake in that part.
As you mentioned, the differences between the previous page and this one are indeed on lines 5 and 17-29 for both the English and Japanese versions.
I will correct the 12 to 5.